superduperdb/backends/base/artifact.py

```diff
@@ -95,18 +101,23 @@ def save_artifact(self, r: t.Dict):
                   and optional fields
                   {'file_id', 'uri'}
         """
-        assert 'bytes' in r, 'serialized bytes are missing!'
-        assert 'datatype' in r, 'no datatype specified!'
-        datatype = self.serializers[r['datatype']]
-        uri = r.get('uri')
-        file_id = None
-        if uri is not None:
-            file_id = _construct_file_id_from_uri(uri)
+        if r.get('type') == 'file':
+            sha1 = hashlib.sha1(str(id(r)).encode()).hexdigest()
```

Comment:
This won't be easily verifiable after adding `r['file_id']` key to `r`. Also 
it will be potentially subject to collisions if the file-name `r['x']` is generic. Like `test.mp3`.
We could hash everything in the directory, but this might be slow and complicated.
As an alternative, we can create a random `file_id`.

Reply:
Let's use random `file_id` now, keep it simple